### PR TITLE
Add aspects to curriculum/javascript/core/errors/

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/errors/exceptions.md?ref=f519d0f0549c09a2c38402852c1df87d2ea64022

File: javascript/core/errors/exceptions.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ links:
 
 
 aspects:
-  - introduction
+  - deep
```
Comment (ryanwarsaw) [inline javascript/core/errors/exceptions.md]: I would revert these changes, it's certainly an `introduction` level insight, and not `obscura`.

File: javascript/core/errors/js-practice-custom-errors.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: https://www.codewars.com/kata/throw-from-list-error-handling-number-3
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/errors/js-practice-custom-errors.md]: Meh, I'd add `introduction` to this too

File: javascript/core/errors/js-practice-error-throwing.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: https://www.codewars.com/kata/error-throwing-error-handling-number-2
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/errors/js-practice-error-throwing.md]: `introduction`

File: javascript/core/errors/js-practice-invalid-input.md [hunk 4]
```diff
@@ -17,6 +17,9 @@ standards:
   javascript.exceptions.1: 1000
   javascript.exceptions.2: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/errors/js-practice-invalid-input.md]: `introduction`, they're likely going to do this a bunch
